inst/www/js/rcloud-flexdashboard.js

```diff
@@ -45,6 +50,9 @@ function main() {
             promise = promise.then(function() {
                 // Just tell R to load the rcloud.flexdashboard package,
                 // and the packag will do the rest
+                if(!rcloud._ocaps.load_module_package) {
+                  throw new Error("Can't load 'rcloud.flexdashboard' - loading of modules is disabled.");
+                }
```

Comment:
This isn't really necessary because it's just a bug in RCloud or this package and it doesn't really matter what error we display.